sdk/communication/Azure.Communication.Chat/src/ChatThreadClient.cs

```diff
@@ -82,7 +120,7 @@ public virtual Response UpdateTopic(string topic, CancellationToken cancellation
             scope.Start();
             try
             {
-                return _chatThreadRestClient.UpdateChatThreadProperties(Id, topic, cancellationToken);
+                return _chatThreadRestClient.UpdateChatThreadProperties(Id, topic, null, null, cancellationToken);
```

Comment:
why are these null?

Reply:
Since this is update topic API. There is "UpdatePropertiesAsync" to allow update metadata, retention policy.